iOS_SDK/OneSignalSDK/Source/OneSignal.m

```diff
@@ -167,9 +167,9 @@ @implementation OneSignal
 // Make sure launchOptions have been set
 // We need this BOOL because launchOptions can be null so simply null checking
 //  won't validate whether or not launchOptions have been set
-static BOOL hasSetLaunchOptions = false;
+static BOOL hasInitWithLaunchOptions = false;
```

Comment:
The name `hasInitWithLaunchOptions` implies to me that launchOptions was `non-nil`.
Could be called `hasCalledInitWithLaunchOptions` but if all we are looking for is `init` was call then thinking we can just keep it simple and call it `hasCalledInit`.

Reply:
I want to distinguish between initDone and this boolean so I will go with `hasCalledInitWithLaunchOptions `